tests/core/test_public_api_surface.py

```
@@ -0,0 +1,171 @@
+    for export_name in kerykeion.__all__:
+        exported = getattr(kerykeion, export_name)
+        if not inspect.isclass(exported):
+            continue
+        for member_name, member in inspect.getmembers(exported):
+            if member_name.startswith("_") or not callable(member):
+                continue
+            function = inspect.unwrap(getattr(member, "__func__", member))
+            try:
+                hints = typing.get_type_hints(function)
+            except Exception:
+                continue
```

Ruff (0.15.15)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)